build.gradle

```diff
@@ -50,14 +51,15 @@ dependencies {
     compile 'io.github.microutils:kotlin-logging:1.4.6'
     
     // Project specific dependencies (disabled by default)
-    compile 'com.github.zensum:franz:59132b791c'
-    compile 'com.github.zensum:webhook-proto:0.1.1'
-    compile 'com.github.zensum:ktor-prometheus-feature:83baca2ca3'
-    compile 'com.github.zensum:ktor-sentry-feature:master-SNAPSHOT'
-    compile 'com.github.zensum:ktor-jwt:0.1.3'
-    compile 'com.github.zensum:ktor-health-check:master-SNAPSHOT'
-    compile 'org.jetbrains.ktor:ktor-core:0.4.0'
-    compile 'org.jetbrains.ktor:ktor-netty:0.4.0'
+    compile 'com.github.zensum:franz:f1e1d92'
+    compile 'com.github.zensum:webhook-proto:d623675'
+    compile 'com.github.zensum:ktor-prometheus-feature:abaa126'
+    compile 'com.github.zensum:ktor-sentry-feature:eece542'
+    compile 'com.github.zensum:ktor-jwt:5dc52cb'
+    compile 'com.github.zensum:ktor-health-check:94a7b02'
+    compile 'io.ktor:ktor-server-core:0.9.1'
+    compile 'io.ktor:ktor-server-netty:0.9.1'
+    testCompile "io.ktor:ktor-server-test-host:0.9.1"
     compile 'com.moandjiezana.toml:toml4j:0.7.2'
     compile 'com.github.rholder.fauxflake:fauxflake-core:1.1.0'
```

Comment:
Replace with `compile 'com.github.jonross:fauxflake:90abbcf5b6'` so we use forked version without bug.